rviz_rendering/src/rviz_rendering/render_window_impl.cpp

```diff
@@ -175,11 +175,11 @@ createScene(Ogre::SceneManager * ogre_scene_manager)
   // sphereMaterial->setSelfIllumination(0.2f, 0.2f, 0.1f);
 
   sphereMesh->setMaterialName("SphereMaterial");
-  childSceneNode->setPosition(Ogre::Vector3(0.0f, 0.0f, 0.0f));
+  childSceneNode->setPosition(Ogre::Vector3(0.0f, 0.5f, 0.0f));
```

Comment:
Why this change?

Reply:
It made the sphere sit up on the mesh ... but this can be discarded.

Reply:
Yeah, I think this code is now obsolete, since we're loading the scene based on the display plugins.